catt/controllers.py

```diff
@@ -146,42 +147,36 @@ def __init__(self, device_name):
 
         self.cast.wait()
 
-    def _check_state(self):
-        if self.cast.app_id == "E8C28D3C" or not self.cast.app_id:
-            raise CattCastError("Chromecast is inactive.")
-        self.cast.media_controller.block_until_active(1.0)
-        if self.cast.media_controller.status.player_state in ["UNKNOWN", "IDLE"]:
-            raise CattCastError("Nothing is currently playing.")
+        if state_check:
```

Comment:
It's generally better to have code that does one thing as a separate function, so they're more self-contained and better documented. If you don't mind changing it, that'd be great, but if you can't be bothered, this is fine as well.

I used the `kill` command instead of the `stop` command because the latter didn't go back to the main Chromecast screen, and I think that would be more intuitive for the user. What's your opinion?